docs/quick-guide.md

```diff
@@ -1,101 +1,122 @@
 ---
-layout: default
-title: Quick Guide
-nav_order: 2
+title: Quick guide
 ---
 
-# Quick Guide
+# Quick guide
 
-## ES5 UMD Bundle
+## npm package
 
-Download the UMD bundle from the
-[latest GitHub release](https://github.com/microbit-foundation/microbit-fs/releases/latest)
-and add it to the page:
+You can integrate this library in your project using the [npm package](https://www.npmjs.com/package/@microbit/microbit-fs):
 
-```html
-<script src="microbit-fs.umd.min.js"></script>
+```bash
+$ npm add @microbit/microbit-fs
```

Comment:
This will work but add is just an alias for install. Should we use `install` instead?